README.md

```diff
@@ -60,6 +60,9 @@ be found at [https://hexdocs.pm/mozart_fetcher](https://hexdocs.pm/mozart_fetche
 
 ## Run locally
 
+You'll need to set the environment variable `DEV_CERT_PEM` to the path to your BBC developer
```

Comment:
```suggestion
You'll need to set the environment variable `DEV_CERT_PEM` to the path of your BBC developer
```

Reply:
I suppose it depends if you consider the path a property of the file itself, or the filesystems way of finding the file ;)